scripts/nopticon.py

```diff
@@ -0,0 +1,48 @@
+"""
+Python classes for Nopticon 
+"""
+
+import ipaddress
+import json
+
+class NetworkSummary:
+    def __init__(self, summary_json):
+        self._summary = json.loads(summary_json)
+
+        # Find flows
+        self._edges = {}
+        for key in self._summary.keys():
```

Comment:
Does this only return the highest-level keys? I would like to double check that what you are capturing is intended. That is, Nopticon returns "flows" and "network-summary" separately, and I don't fully understand yet which of these two you'd like to parse here.

Reply:
Perhaps the term "prefixes" would be more appropriate than flows. 

For example, consider the last network-summary produced for ft4. It contains the following keys: `['3.0.0.0/24', 'network-summary', 'flows', '3.1.0.0/24', '3.3.0.0/24', '3.2.1.0/24', '3.0.1.0/24', '3.1.1.0/24', '3.2.0.0/24', '3.3.1.0/24']`
I am looking for all of the prefix keys (e.g., 3.0.0.0/24, 3.1.0.0/24, etc.), because the value for each of them is a list of edges that have that prefix label.

It would be more convenient if there was a top-level key "edges" (or another meaningful name), whose value was a map from prefixes (e.g., 3.0.0.0/24, 3.1.0.0/24, etc.) to a list of edges with that prefix label. For example, the resulting JSON would be something like:
```
{
  "network-summary" : "flow",
  "edges" : {
    "3.0.0.0/24" : [
        { "source" : "core0", "target" : "core3", "rank-0" : 0.9999899864196777 },
        ...
    ],
    "3.1.0.0/24" : [
        { "source" : "core0", "target" : "agg1_0", "rank-0" : 0.9999899864196777 },
        ...
    ],
    ...
  }
  "flows" : [
    {"ranges" : [ ... ], "flow" : "3.0.0.0/24", "links" : [ ... ]},
    ...
  ]
}
```